### DISCUSSION TITLE
I'm getting an error when trying to build a package for fumadocs

### DISCUSSION THREAD

**Discussion by densumesh** [fuma-nama/fumadocs#1003] [General]
I am able to get the build running, but when i click on the search component to render my custom search dialog, I am getting this error: 
![image](https://github.com/user-attachments/assets/c47c103c-14f5-46c7-918d-0d127d558c41)

My package is linked here: https://github.com/devflowinc/trieve/pull/2665

**Comment by fuma-nama** [comment]
You have to wrap components inside https://fumadocs.vercel.app/docs/ui/blocks/root-provider
